assets/app/scripts/filters/resources.js

```diff
@@ -28,7 +38,25 @@ angular.module('openshiftConsole')
         }
       }
       if (resource && resource.metadata && resource.metadata.annotations) {
-        return resource.metadata.annotations[key];
+        // If the key's already in the annotation map, return it.
+        if (key in resource.metadata.annotations) {
```

Comment:
my js prototypal inheritance is rusty... was just thinking `if (resource.metadata.annotations[key] !== undefined) {...`